src/Source/Source.js

```diff
@@ -16,7 +16,7 @@ export const supportedFetchers = new Map([
     ['application/json', Fetcher.json],
     ['application/kml', Fetcher.xml],
     ['application/gpx', Fetcher.xml],
-    ['application/x-protobuf;type=mapbox-vector', Fetcher.arrayBuffer],
+    ['application/x-protobuf;type=mapbox-vector', Fetcher.mapbox],
```

Comment:
same
Fetcher.multisource that can recognize arrayBuffer
OR
Fetcher.arrayBuffer that can deal with multisource  ?